tests/jsx/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,5 +1,84 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`conditional-expression.js 1`] = `
+<div>
+  {isCool ? "cool" : "very nice"}
+</div>;
+
+// should get formatted JSX style
+const a = something ? "string" : <div />;
+
+// should not get formatted JSX style
+const a = something ? "string" : "other string";
+
+<div>
+  {reallyLongChainOfConditionalExpressions ? (
+    "why"
+  ) : yes ? (
+    "it"
+  ) : is ? (
+    "!"
+  ) : (
+    ":)"
+  )}
+</div>;
+
+<div>
+  {reallyLongChainOfConditionalExpressions ? (
+    <span>why</span>
+  ) : yes ? (
+    <span>it</span>
+  ) : is ? (
+    <span>!</span>
+  ) : (
+    <span>:)</span>
+  )}
+</div>;
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<div>
+  {isCool ? (
```

Comment:
Personally, I'm not so sure that the #2208 workflow is needed for non-JSX literals. The parens around string and null literals seem quite ugly to me, and not something likely to ever be written by a human voluntarily 😄 

In particular, `{isCool ? "cool" : "very nice"}` seems far better than what we have here, and isn't uncommon in actual code. 

That said, there would be some tricky consistency issues, eg; 

```jsx
{showTheThing ? (
  <Thing />
) : null}
```
seems awkward/inconsistent, though 
```jsx
{showTheThing ? (
  <Thing />
) : (
  null
)}
```
seems bloated/verbose